mslib/msui/multiple_flightpath_dockwidget.py

```diff
@@ -240,3 +243,24 @@ def remove_item(self):
                 self.labelStatus.setText("Status: FTML File is Removed")
         if not flag:
             self.labelStatus.setText("Status: Select FTML File to Delete")
+
+    def syncFlighttrackLists(self):
+        """
+        Add Flighttracks from MSUI ListView to multiple_flightpath_dockwidget list
+        """
+        for index in range(self.listView.count()):
+            item = self.listView.item(index).text()
+            # if item not in self.list_flighttrack:
+            self.create_list_item(item)
+
+    def test(self):
```

Comment:
test is not a good name, if it is a pytest test maybe add it to the _test dir. If  it is something else name it different.

Reply:
Changed